deployment/terraform/database_anonymizer_sheduled_task/docker/database_anonymizer.py

```diff
@@ -113,7 +113,13 @@
 )
 
 waiters = source.get_waiter('db_snapshot_completed')
-waiters.wait(DBSnapshotIdentifier=shared_snapshot_identifier)
+waiters.wait(
+    DBSnapshotIdentifier=shared_snapshot_identifier,
+    WaiterConfig={
+        'Delay': 15,
```

Comment:
Can we use something like AWS EventBridge instead of relying of delay of 15 seconds?

Reply:
No, we must wait until the snapshot copy is complete before proceeding with the next steps.

Reply:
Ack.